src/index.html

```diff
@@ -7,8 +7,31 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="logo"></a>
+      <nav class="nav">
+        <ul class="ul">
+          <a href="" class="nav__list is-active">Apple</a>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="" class="nav__link is-active">Apple</a>
```
Всередині тега `ul` мають знаходитьсь теги `li` (list items), а не будь які інші. Зараз в тебе не валідний код.
Всередині `li` вже можуть знаходитись інші, будь які теги